lib/internal/modules/esm/module_job.js

```diff
@@ -140,7 +140,7 @@ class ModuleJob {
           /module '(.*)' does not provide an export named '(.+)'/,
           e.message);
         const { url: childFileURL } = await this.loader.resolve(
-          childSpecifier, parentFileUrl,
+          childSpecifier, parentFileUrl, kEmptyObject,
```

Comment:
nit, to avoid future diff churn:
```suggestion
          childSpecifier,
          parentFileUrl,
          kEmptyObject,
```